### PR TITLE
Created CSS

### PR DESCRIPTION
**Pull request by hd5420**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -4,32 +4,47 @@
...
+</div>
+<a src="index.html"><img src="images/me.JPG" alt="Open book and a cup of coffee."> </a>
 </main>
 <footer>
-        <h2>-Disclaimer-</h2>
-        <p>The content on this site belongs to me unless otherwise noted. If using any information found here, please give credit where credit is due. The views expressed are those of myself and not representative of Wayne State University or my employer.</p>
+    <ul>
+        <li><a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" class= "cc"/></a> 
+    <a href="https://twitter.com/almemoore">
+        <img src="images/twitter.png" class="tw">
```
Comment (axamei) [inline about.html]: The logos need alternative text. Add `alt` attribute to each img

File: about.html [hunk 2]
```diff
@@ -4,32 +4,47 @@
...
 </header>
+ <body>
 <main>
     <h1>About Me</h1> 
+    <div>
     <img src= "images/banned.jpeg" alt= "Person standing behind a stack of books." >
-    <h3><ul>
+</div>
+<div>
+<h3>
```
Comment (axamei) [inline about.html]: A heading level was skipped. You went from `h1` to `h3`. Think of headings like a table of contents for the page content. If you are trying to style the list to have a certain font size, use CSS properties like `font-size`.

File: connect.html [hunk 3]
```diff
@@ -4,19 +4,18 @@
...
+    <header>
+        <nav>
+            <a href= "index.html"><img src="images/ALMLogo.png" class= "logo" alt="Open Book with flowers and Angela L. Moore written underneath."></a>
+               <a href="about.html">About</a>
+                <a href="work.html">Work</a>
+                <a href="connect.html">Connect</a>
+        </nav>
+    </header>
+    <main>
    <h2>Where to Find Me</h2>
```
Comment (axamei) [inline connect.html]: You are missing an `h1` on this page.

File: index.html [hunk 4]
```diff
@@ -3,29 +3,39 @@
...
 <main>
-    <div><img src="images/ignorance.jpeg" alt="Person wearing a shirt that says Reading can seriously damage your ignorance"></div>
-    <div>
-    <h1 id="section-title"> About Me</h1>
-        <p>I am currently a student in the School of Information Science at Wayne State Univeristy, working on the MLIS with a school library focus.  I am also living my best life as an elementary school library media specialist teaching Kindergarten through 5th Grade. </p> 
-   </div>
+    <body>
+    <div class="featured-image-container"><img src="images/stack.jpg" alt="Stack of picture books."></div>
+    <div class="featured-image-container">
+    <h2 id="section-title">Introduction</h2>
```
Comment (axamei) [inline index.html]: You are missing an `h1` on this page

**Review by axamei — CHANGES_REQUESTED** [review]
Please add alt text to your logo images on all HTML pages and make sure you are not skipping heading levels.

**Comment by hd5420** [comment]
Hi @axamei I fixed these last week and submitted a pull request. I hope I didn't mess up again! :) Thanks!! ~Angie

**Review by axamei — APPROVED** [review]
👍

**Comment by axamei** [comment]
Hi @hd5420 I merged this work into the main branch. Book an appointment if you would like a quick walkthrough of the next steps

**Comment by hd5420** [comment]
Thank you! I'll see you in the morning!

**Pull request merged**